schema/bidder_position.js

```diff
@@ -45,6 +45,15 @@ const BidderPositionType = new GraphQLObjectType({
       type: SaleArtwork.type,
       resolve: position => gravity(`sale_artwork/${position.sale_artwork_id}`),
     },
+    is_winning: {
```

Comment:
Not sure if this it the best way to go about this. This field would mimic [the Force PR logic](https://github.com/artsy/force/pull/4284/files#diff-e13e2d4daed3c7935d99823410845942R4).

It seems silly if this fetches the sale artwork twice when used in combo with `sale_artwork`, e.g.

````
query {
  me {
    bidder_positions(current: true) {
      id
      is_winning
      sale_artwork {
        id
      }
    }
  }
}
````

Although I wonder if that's a bid deal depending on how the caching works. I also wonder if there's some kind of currying/lazy-loading thing we could put in `gravity` that can recognize the same query being made twice at the same time and make only one fetch.

Curious to hear your thoughts on this implementation.